### PR TITLE
Create 137_single_number.py

### PR DESCRIPTION
**Pull request by jsingh41**
@sb2nov

### CODE REVIEW

**Comment by jsingh41** [comment]
This does use extra memory though no?

**Comment by sb2nov** [comment]
Yes it uses O(N) memory while the sum case didn't do that.

File: 137_single_number.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Your algorithm should have a linear runtime complexity. Could you implement it without using extra memory?
+"""
+
+class Solution(object):
+    def singleNumber(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        # For no extra memory, we'll go with the pop method
```
Comment (sb2nov) [inline 137_single_number.py:15]: Even the pop method has some memory usage. The sum method is the most memory efficient.

**Review by sb2nov — APPROVED** [review]
LGTM, please merge

**Pull request merged**